content/en/blog/2025/opentelemetry-logging-and-you.md

```diff
@@ -0,0 +1,93 @@
+---
+title: OpenTelemetry Logging and You
+linkTitle: OTel Logging
+date: 2025-04-16
+author: >-
+  [Austin Parker](https://github.com/austinlparker) (honeycomb.io)
+sig: Logs
+---
+
+If you’ve been following OpenTelemetry for a while, you’ve probably heard a lot
+about logs. [Log bridges](/docs/specs/otel/glossary/#log-appender--bridge),
+[Logs API](/docs/specs/otel/logs/api), [Events](/docs/concepts/glossary/#event),
+you name it, we’ve talked about it. This blog post is intended to be a
+discussion of the rationale and current design direction of logging in
+OpenTelemetry.
+
+## Definitions
+
+Let’s get started with a basic definition of how OpenTelemetry thinks about
+logs. Broadly, logs are any telemetry that is emitted through a LoggerProvider,
```

Comment:
LoggerProvider feels like an overly technical word for this post, maybe just "logs are any telemetry that is emitted through the Logs API"? although that sounds tautological